client/image_build.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/docker/docker/api/types/container"
 )
 
-// ImageBuild sends request to the daemon to build images.
-// The Body in the response implement an io.ReadCloser and it's up to the caller to
+// ImageBuild sends a request to the daemon to build images.
+// The body in the response implements an io.ReadCloser and it's up to the caller to
```

Comment:
I think in this case, `Body` refers to the `Body` field in the response; https://github.com/moby/moby/blob/46cdcd206c56172b95ba5c77b827a722dab426c5/client/image_build.go#L42-L45